lib/internal/Magento/Framework/TestFramework/Unit/Listener/GarbageCleanup.php

```diff
@@ -5,98 +5,23 @@
  */
 namespace Magento\Framework\TestFramework\Unit\Listener;
 
-use PHPUnit\Framework\Test;
-use PHPUnit\Framework\Warning;
+use PHPUnit\Framework\TestListener;
+use PHPUnit\Framework\TestSuite;
 
 /**
  * Listener of PHPUnit built-in events that enforces cleanup of cyclic object references
  *
  */
-class GarbageCleanup implements \PHPUnit\Framework\TestListener
+class GarbageCleanup implements TestListener
 {
-    /**
-     * {@inheritdoc}
-     * @SuppressWarnings(PHPMD.ShortVariable)
-     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
-     */
-    public function addError(\PHPUnit\Framework\Test $test, \Exception $e, $time)
-    {
-    }
-
-    /**
-     * {@inheritdoc}
-     * @SuppressWarnings(PHPMD.ShortVariable)
-     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
-     */
-    public function addFailure(\PHPUnit\Framework\Test $test, \PHPUnit\Framework\AssertionFailedError $e, $time)
-    {
-    }
-
-    /**
-     * {@inheritdoc}
-     * @SuppressWarnings(PHPMD.ShortVariable)
-     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
-     */
-    public function addIncompleteTest(\PHPUnit\Framework\Test $test, \Exception $e, $time)
-    {
-    }
-
-    /**
-     * {@inheritdoc}
-     * @SuppressWarnings(PHPMD.ShortVariable)
-     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
-     */
-    public function addRiskyTest(\PHPUnit\Framework\Test $test, \Exception $e, $time)
-    {
-    }
-
-    /**
-     * {@inheritdoc}
-     * @SuppressWarnings(PHPMD.ShortVariable)
-     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
-     */
-    public function addSkippedTest(\PHPUnit\Framework\Test $test, \Exception $e, $time)
-    {
-    }
-
-    /**
-     * {@inheritdoc}
-     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
-     */
-    public function startTestSuite(\PHPUnit\Framework\TestSuite $suite)
-    {
-    }
+    use \PHPUnit\Framework\TestListenerDefaultImplementation;
 
     /**
      * {@inheritdoc}
```

Comment:
This is correct finding.
`20 | ERROR | [x] If the @inheritdoc not inline it shouldn’t have braces`